src/entt/entity/storage.hpp

```diff
@@ -425,16 +535,21 @@ class basic_storage: public basic_sparse_set<Entity, typename std::allocator_tra
     /*! @brief Constant extended iterable storage proxy. */
     using const_iterable = iterable_adaptor<internal::extended_storage_iterator<typename base_type::const_iterator, const_iterator>>;
 
+    using underlying_type::type;
```

Comment:
??

Reply:
Required for deduplication.

Reply:
Can you elaborate a little further? I think I'm missing some context here.